S/seatd/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "seatd"
+version = v"0.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://git.sr.ht/~kennylevinsen/seatd/archive/$version.tar.gz", "274b56324fc81ca6002bc1cdd387668dee34a6e1063e5f3896805c3770948988"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/seatd-*/
+atomic_patch ../patches/fix_crosscompile.patch
+mkdir build
+cd build
+apk add meson scdoc
+
+PKG_CONFIG_SYSROOT_DIR="" meson ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
+mkdir ${includedir}
```

Comment:
Why is this needed?

Reply:
It somehow wasn't created automatically, so ninja errored saying the directory does not exist.

Reply:
I'm asking because the directory is created for me (and I would be very surprised otherwise)

Reply:
I've tried only 1 or 2 platforms, if that matters

Reply:
It still fails for me, weird... Let's try what happens without it

Reply:
Seems to work here. Any ideas why this could be? Should I open a separate issue?

Reply:
I'd need to see _what_ is the error and a way to reproduce it.  But I have a hard time imaging it's a bug in BinaryBuilder, if that's what you're suggesting.  My guess would be a bug in either meson or the build system of the package, I don't see why a missing `${includedir}` would be a problem and a build system which doesn't automatically create a destination directory (if that was the problem) would be thoroughly broken.

Reply:
Ah, it's a compiler warning:
```
[23:12:26] ninja: job failed: /opt/bin/x86_64-linux-gnu-libgfortran4-cxx11/x86_64-linux-gnu-gcc -Ipoller_test@exe -I. -I.. -I../include -I/workspace/destdir/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Wpedantic -Werror -std=c11 -g -Wundef -Wunused -Wlogical-op -Wmissing-include-dirs -Wold-style-definition -Wpointer-arith -Wstrict-prototypes -Wimplicit-fallthrough -Wmissing-prototypes -Wno-unknown-warning-option -Wno-unused-command-line-argument -Wvla -Wl,--exclude-libs=ALL -D_XOPEN_SOURCE=700 -D__BSD_VISIBLE '-DSEATD_VERSION="0.5.0"' '-DSEATD_DEFAULTPATH="/run/seatd.sock"' -DDEBUG '-DREL_SRC_DIR="../"' -DSEATD_ENABLED=1 -MD -MQ 'poller_test@exe/common_linked_list.c.o' -MF 'poller_test@exe/common_linked_list.c.o.d' -o 'poller_test@exe/common_linked_list.c.o' -c ../common/linked_list.c
[23:12:26] cc1: error: /workspace/destdir/include: No such file or directory [-Werror=missing-include-dirs]
[23:12:26] cc1: error: unrecognized command line option ‘-Wno-unused-command-line-argument’ [-Werror]
[23:12:26] cc1: error: unrecognized command line option ‘-Wno-unknown-warning-option’ [-Werror]
[23:12:26] cc1: all warnings being treated as errors
```

Reply:
Not sure why my compiler would behave differently here though

Reply:
Oh, I see they explicitly add the flag `-Wmissing-include-dirs`: https://git.sr.ht/~kennylevinsen/seatd/tree/master/item/meson.build#L30, but then the build system does add missing include directories... :shrug: